application/cms/models.py

```diff
@@ -102,18 +102,83 @@ def process(value):
 
 
 class FrequencyOfRelease(db.Model):
+    __tablename__ = "frequency_of_release"
+
     id = db.Column(db.Integer, primary_key=True)
     description = db.Column(db.String(), nullable=False)
     position = db.Column(db.Integer, nullable=False)
 
 
 class TypeOfStatistic(db.Model):
+    __tablename__ = "type_of_statistic"
+
     id = db.Column(db.Integer, primary_key=True)
     internal = db.Column(db.String(), nullable=False)
     external = db.Column(db.String(), nullable=False)
     position = db.Column(db.Integer, nullable=False)
 
 
+class DataSource(db.Model):
+    __tablename__ = "data_source"
+
+    # columns
+    id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String(255), nullable=True)
+
+    type_of_data = db.Column(ArrayOfEnum(db.Enum(TypeOfData, name="type_of_data_types")), default=[], nullable=True)
+    type_of_statistic_id = db.Column(db.Integer, nullable=True)
+
+    publisher_id = db.Column(db.String(255), nullable=True)
```

Comment:
@samuelhwilliams should we call this `organisation_id` to match the table & model name?

Reply:
I think `publisher_id` is clearer about what the relationship signifies. `organisation_id` doesn't really tell you how the organisation is related to the source.

Reply:
Good point. Let’s stick with publisher.